test/test.hs

```diff
@@ -17,11 +17,24 @@ type UnitPrefix (n :: Nat) = Eval (Guarded n
    , Otherwise ':= Error "Something else"
    ])
 
+type UnitPrefix' = Case [
```

Comment:
List here (and in docs) without tick seems nicer, should be always resolved right because of kind and doesn't seem to be problem even with `-Wall`.

Reply:
Fair enough. Could you also remove the tick from the other test above?

I guess the tick would be necessary for 0 or 1 elements, but that probably won't happen with this construct.